packages/core/src/zip/__tests__/elementZipper.js

```diff
@@ -131,69 +136,72 @@ describe('Zipper', () => {
     ],
   }
 
-  it('zipper should correctly navigate up down left and right', () => {
+  it.skip('zipper should correctly navigate up down left and right', () => {
```

Comment:
🤔 I think we need to check what the problem is here. ping @ognen @AleksandarDembovski